src/api/_mocks/characters/id/get/mock.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import fetchMock from 'fetch-mock';
+/* eslint-disable import/prefer-default-export */
```

Comment:
Echoing the same two bullet points from the prior comment.

Reply:
Same answer others two bullets.